docs/v2.5/quick-start/index.md

```diff
@@ -1,7 +1,186 @@
 ---
-   description: Telepresence Quick Start.
+description: "Start using Telepresence in your own environment. Follow these steps to intercept your service in your cluster."
 ---
 
-import TelepresenceQuickStartLanding from './TelepresenceQuickStartLanding';
+import Alert from '@material-ui/lab/Alert';
+import Platform from '@src/components/Platform';
+import QSCards from '../quick-start/qs-cards'
 
-<TelepresenceQuickStartLanding/>
+# Intercept a service in your own environment
+
+Telepresence enables you to create intercepts to a target Kubernetes workload. Once you have created and intercept, you can code and debug your associated service locally.
+
+For a detailed walk-though on creating intercepts using our sample app, follow the [quick start guide](../../quick-start/demo-node/).
+
+## Prerequisites
+
+Before you begin, you need to have either the Kubernetes command-line tool, [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or the OpenShift Container Platform command-line interface, [`oc`](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands). This document uses kubectl in all example commands. OpenShift users can substitute oc [commands instead](https://docs.openshift.com/container-platform/4.1/cli_reference/developer-cli-commands.html).
+
+This guide assumes you have a Kubernetes deployment and service, and that you can run a copy of that service on your laptop.
+
+## Install telepresence on your machine
+
+Install Telepresence by running the commands below for your OS. If you are not the administrator of your cluster, you will need [administrative RBAC permissions](../reference/rbac#administrating-telepresence) to install and use Telepresence in your cluster.
+
+<Platform.TabGroup>
+<Platform.MacOSTab>
+
+```shell
+# Intel Macs
+# Install via brew:
+brew install datawire/blackbird/telepresence
+# OR install manually:
+# 1. Download the latest binary (~60 MB):
+sudo curl -fL https://app.getambassador.io/download/tel2/darwin/amd64/$dlVersion$/telepresence -o /usr/local/bin/telepresence
+# 2. Make the binary executable:
+sudo chmod a+x /usr/local/bin/telepresence
+# Apple silicon Macs
+# Install via brew:
+brew install datawire/blackbird/telepresence-arm64
+# OR Install manually:
+# 1. Download the latest binary (~60 MB):
+sudo curl -fL https://app.getambassador.io/download/tel2/darwin/arm64/$dlVersion$/telepresence -o /usr/local/bin/telepresence
+# 2. Make the binary executable:
+sudo chmod a+x /usr/local/bin/telepresence
+```
```

Comment:
this is 4 different scripts, not one script, they should not all be copied by the same button

Reply:
Do you have an example for this? I found this one, but copy all the text: 

![Screen Shot 2022-10-13 at 9 53 08](https://user-images.githubusercontent.com/101214760/195631452-d248515c-cb0d-434a-b4a8-2d3f149a6192.png)

Reply:
i dont have an example. maybe make four different shell boxes?